src/nvim/terminal.c

```diff
@@ -618,6 +621,17 @@ static int term_movecursor(VTermPos new, VTermPos old, int visible,
   return 1;
 }
 
+// Set the terminal title. Takes ownership of the title string.
```

Comment:
As far as I can tell, the string is placed into the dictionary uncopied, and when strings are subsequently removed from that dictionary, they're freed.  Have I missed a copying function somewhere?